packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -1006,6 +994,107 @@ export class AppStudioPluginImpl {
     return ok({ tabEndpoint, tabDomain, aadId, botDomain, botId, webApplicationInfoResource });
   }
 
+  private getTabEndpoint(ctx: PluginContext, isLocalDebug: boolean) {
```

Comment:
it's better to announce the return type?

Reply:
Good catch! Let's me fix it.